webplugin/js/app/kommunicate-ui.js

```
@@ -1061,50 +930,32 @@ KommunicateUI = {
-                        id: ['csat-1'],
-                    },
-                    'n-vis'
-                );
+                kommunicateCommons.hide({
+                    class: ['mck-ratings-smilies'],
+                });
+                kommunicateCommons.hide({
+                    id: ['csat-1'],
+                });
             };
             var isCSATenabled = kommunicate._globals.oneTimeRating
```

Biome (2.1.2)

[error] Shouldn't redeclare 'isCSATenabled'. Consider to delete it or rename it.

'isCSATenabled' is defined here:


(lint/suspicious/noRedeclare)